File: 1_class_object.cpp
#include<bits/stdc++.h>
using namespace std;

class A
{
    public:
    int value;

    A(int value)
    {
        this->value = value;
        cout<<"I am construtor"<<endl;
    }

    int Display()
    {
        return value;
    }
    ~A()
    {
        cout<<"I am distructor"<<endl;
    }
};
int main()
{
    A obj(10);
    cout<<obj.Display()<<endl;
    return 0;
}

File: 1_class_object.py

class A:
    def __init__(self):
        self.value = 10
        print("I am constructor")

    def display(self):
        print("I am Method")

    def __del__(self):
        print("I am destructor")

obj = A()
obj.display()
# print(obj.value)
obj.value_1 = 20
print(obj.__dict__)